part1/chapter7/quiz1/questions/question_2.rb

```diff
@@ -50,4 +50,26 @@
 #   many groups they want to create.  You can assume the user will
 #   input integers for group numbers that exist when they are asked
 #   for the number of a group to print out.
+def get_groups
+  groups_array = []
+  puts "How many groups would you like to create? "
+  groups = gets.chomp
+  groups_array << groups
```

Comment:
This will give you an array containing one element: the user's answer.
```ruby
[] << "5"
# => ["5"]
```
I don't *think* that's what you intended. Possibly caused by naming the variable `groups` rather than e.g. `number_of_groups`?